week-3-tests/test_dataset.py

```diff
@@ -0,0 +1,41 @@
+from train import get_dataset
+import pandas as pd
+import great_expectations as ge
+import os
+
+
+# create pandas dataset for test set from huggingfaces format
+DATASET_DIR = "./dataset/"
+DATASET_NAME = "imdb.csv"
+if not os.path.exists(DATASET_DIR):
+    os.mkdir(DATASET_DIR)
+if not os.path.exists(DATASET_DIR+DATASET_NAME):    
+    dataset = get_dataset()
+    df = pd.DataFrame(dataset["train"])
+    df.to_csv(DATASET_DIR+DATASET_NAME, index=False)
+
+df = pd.read_csv(DATASET_DIR+DATASET_NAME)
+df = ge.from_pandas(df)
```

Comment:
please use [fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) for this

Reply:
done